src/scopes/goals/reportText.js

```diff
@@ -0,0 +1,10 @@
+import { Op } from 'sequelize';
+
+// TODO: Hmmmmm
```

Comment:
IMO - I think it's totally fine to just remove the report text filter from the goals dashboard

Reply:
same, I removed this filter from the frontend but left the stub in `src/scopes/goals/reportText.js` in case we decide later this filter makes sense in the context of goals